spec/models/address_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Address, type: :model do
+  describe "associations" do
+    it { is_expected.to belong_to(:user) }
+    it { is_expected.to belong_to(:order) }
+  end
+
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:street_number) }
+    it { is_expected.to validate_presence_of(:street_name) }
+    it { is_expected.to validate_presence_of(:city) }
+    it { is_expected.to validate_presence_of(:state) }
+    it { is_expected.to validate_presence_of(:zip) }
+    it { is_expected.to validate_presence_of(:user) }
+    it { is_expected.to validate_presence_of(:order) }
```

Comment:
I'm a little surprised to see this validation. Can users have addresses that aren't used on an order?